### ISSUE TITLE
CLI plugins with HTTP protocol option not being recognized

### ISSUE THREAD

**Issue by JillieBeanSim** [zowe/zowe-cli#1329]
in Zowe CLI v 6.30.0 plugins could have their own protocol option for connection to be HTTP, any version above that starting with 6.31.0 to the present 6.37.7 when a user sets the protocol for the plugin to HTTP the following error is returned

![image](https://user-images.githubusercontent.com/49491949/156415771-64d63144-dd94-436f-8957-2129585600a2.png)

not sure what changed between 6.30.0 and 6.31.0 for this error to occur.

**Comment by zFernand0** [comment]
Hey @JillieBeanSim,
After trying unsuccessfully to reproduce the issue, we believe it might be best to get together in order to find a suitable solution.

**Comment by ItsKipz** [comment]
Hello,

I am experiencing this same issue, but it is happening on every connection regardless of whether i set it to HTTP or HTTPS.

**Comment by phaumer** [comment]
@zFernand0 would be better to get rid of the protocol option in Zowe profiles altogether? I think this could be done for v3.

**Comment by zFernand0** [comment]
> zFernand0 would be better to get rid of the protocol option in Zowe profiles altogether? I think this could be done for v3.

We could consider something like this.
Question for you though,
Would you expect to pass a `<protocol>://<hostname or ip>` combination on instead of just the host?

@adam-wolfe may help us determine whether we can still include this in the V3 plan 😋

**Comment by phaumer** [comment]
@zFernand0 no, I would stick with the host and port properties and only remove protocol. Therefore, default Zowe CLI would only support https for z/OSMF. Other plugins can add other properties for whatever they want to support, e.g. ftp might bring protocol back for its own profiles to distinguish sftp from ftps.
